src/common/store/background-epics.ts

```diff
@@ -20,19 +20,21 @@ import {
 import dashboardMessaging from 'background/dashboard-messaging';
 import { selectUser } from 'common/store/storage/selectors';
 
-interface FluxStandardAction {
+export interface FluxStandardAction {
   type: string | symbol | any;
   payload?: any;
   error?: boolean | any;
   meta?: any;
 }
 
-export const locateEpic: Epic<FluxStandardAction> = action$ => action$.pipe(
+export type PPEpic = Epic<FluxStandardAction, FluxStandardAction, IState>
```

Comment:
If it is general util,so let's call it in general way and reserve `PP` prefix for generating some unique namespace etc. What do you think about defining such cases of usage?

Reply:
Since this type is specific for our project (using `IState` I thought it was not universal enough). I'm thinking now of `StandardEpic`. What do you think?

Reply:
Yep, much better :)